maintainers/maintainer-list.nix

```diff
@@ -1353,6 +1353,11 @@
     github = "domenkozar";
     name = "Domen Kozar";
   };
+  doronbehar = {
+    email = "me@doronbehar.com";
+    github = "doronbehar";
+    name = "Doron Behar";
+  };
```

Comment:
You had it correct earlier, this should remain in a separate commit. Also PR title needs updating, thx!

Reply:
The two separate commits I've had earlier were: 1. maintainer-list and new pdfcpu/default.nix and 2. Update license.

I've uploaded a forced push of first the maintainer update and then the `pdfcpu/default.nix` addition.